app.ts

```diff
@@ -0,0 +1,130 @@
+"use strict";
+window.onload = () => {onloadValues() };
+
+const rowHeight = 56;
+let from = 0;
+let to = 0;
+
+
+function amountOfRowsThatFit(){
+	let $buttonEl = $("button");
+
+	let windowHeigt = window.innerHeight;
```

Comment:
All fields that do not get changed within their scope should be `const`, objects and arrays should be `const`s in most cases too

Reply:
Also, pls correct the spelling of _Height_